xtable-aws/src/test/java/org/apache/xtable/glue/TestGlueCatalogSyncClient.java

```diff
@@ -75,7 +76,12 @@ public class TestGlueCatalogSyncClient extends GlueCatalogSyncTestBase {
 
   private GlueCatalogSyncClient createGlueCatalogSyncClient() {
     return new GlueCatalogSyncClient(
-        catalogConfig, testConfiguration, mockGlueCatalogConfig, mockGlueClient, mockTableBuilder);
+        catalogConfig,
+        testConfiguration,
+        mockGlueCatalogConfig,
+        mockGlueClient,
+        mockTableBuilder,
+        Optional.empty());
```

Comment:
Similar to the Hive PR, let's add similar parameterized testing around this option